index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="Home">
+      <nav id="top-nav">
+        <img src="./images/facebook.svg" alt="">
+        <img src="./images/twitter.svg" alt="">
+        <h6>English</h6>
+        <h6>My page</h6>
+        <h6>Logout</h6>
+      </nav>
+    <div class=" flex nav-bar">
+        <a href="#Home"><img src="./images/gtco_logo2.png" alt="Logo"></a>
+        <ul class="flex nav-list-item">
+            <li class="nav-item nav-about-page"><a href="about.html">About</a></li>
+            <li class="nav-item nav-bar-element"><a href="#">Exhibition</a></li>
+            <li class="nav-item nav-bar-element"><a href="#">Join</a></li>
+            <li class="nav-item nav-bar-element"><a href="#">Sponsor</a></li>
+            <li class="nav-item nav-bar-element"><a href="#">Schedules</a></li>
+            <li class="nav-item nav-border"><a href="#program">PROGRAMS</a></li>
+        </ul>
+    </div>
+    </header>
+    
+    <div class="back-ground">
+    
+    <!-- Navigation Menu Mobile -->
+        <nav class="mobile-menu">
+            <a>
+                <button class="open-btn">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
+                    <div class="bar3"></div>
+                </button>
+            </a>
+
+            <div id="menu-items" class="nav-list">
+            <a class="closebtn" href="">&times;</a>
+                <a class="mobile-nav" href="about.html">About</a>
+                <a class="mobile-nav" href="about.html">Contact</a>
+                <a class="mobile-nav" href="#featured-speakers">PROGRAMS</a>
+                <a class="mobile-nav" href="#">Exhibition</a>
+            </div>
+        </nav>
+    
+      <main id="main">
+    <div class="desk">
+      <p class="before-h12">"Hello! Football Lovers"</p>
+      <h1>FOOTBALL ANNUAL FESTIVAL 2022 </h1>
+      <p class="description-container">
+        FOOTBALL MEETUP Fair is a an annual football exhibition and sales event aimed at projecting the diverse angles of the football industry.
+      </p>
+      <h2>2022.04.30 (Wed) ~ 2ND MAY (Sat) <br><p class="paragraph">@ FT TOWER, Cresecent Avenue, meme, lg, Texas.</p></h2>
+      
+    </div>
+    </div>
+    <div class="cover-program">
```

Comment:
- [ ] I would recommend you remove this div, the appropriate tag which is a section tag has followed it, then this generic tag is not needed.